add note for podman rootful mode (#2199)
astronomer/docs: astro/cli/configure-cli.md

> ## Run the Astro CLI using Podman

Reviewer: ```suggestion
## Run the Astro CLI using Podman

```

:::info 

> If you are using M1 Mac, you will need to turn on rootful mode for Podman using `podman machine set --rootful`.

Reviewer 2: ```suggestion
If you use an M1 Mac, turn on rootful mode for Podman by using `podman machine set --rootful`.
```
This is a style suggestion to make the composition active voice - if you're interested in learning more about why we do that in docs, would be happy to share!

Author: by mistake I clicked on the wrong button, making these changes now.
